CSC8503/TutorialGame.cpp

```diff
@@ -178,7 +182,9 @@ void TutorialGame::UpdateGame(float dt) {
 
 	if (mGameObjects.size() > 0) {
 		for (int i = 0; i < mGameObjects.size(); i++) {
-			mGameObjects[i]->UpdateObject(dt);
+			if (mGameObjects[i] != nullptr){
```

Comment:
I think this should be the responsibility of whatever is filling mGameObjects: mGameObjects should probably never decrease in size, so should never contain a nullptr.

Reply:
Yeah I agree but rn due to the "TutorialGame" being parent class of the "DebugNetworkedGame", when parent constructor is called it's adding some objects that not fully initialized or something else. I added this check to debug what was wrong. I will remove it and try to the necessary changes to prevent this as soon as I can. FYI 🦖